packages/wallet/backend/src/gatehub/service.ts

```diff
@@ -69,7 +79,80 @@ export class GateHubService {
 
     await User.query().findById(user.id).patch(userDetails)
 
-    return isUserApproved
+    let customerId
+    if (
+      this.env.NODE_ENV === 'development' &&
+      this.env.GATEHUB_ENV === 'sandbox'
+    ) {
+      customerId = await this.setupSandboxCustomer(
+        user.id,
+        user.gateHubUserId,
+        userState
+      )
+    }
+
+    return { isUserApproved, customerId }
+  }
+
+  private async setupSandboxCustomer(
+    userId: string,
+    managedUserId: string,
+    userState: IGetUserStateResponse
+  ): Promise<string> {
+    if (!this.accountService || !this.walletAddressService) {
+      throw new Error(
+        'AccountService and WalletAddressService must be provided in sandbox environment.'
+      )
+    }
+
+    const account = await this.accountService.createDefaultAccount(
+      userId,
+      'EUR Account',
+      true
+    )
+    if (!account) {
+      throw new Error('Failed to create account for managed user')
+    }
+
+    const paymentPointer = await createWalletAddressIfFalsy({
+      userId,
+      accountId: account.id,
+      publicName: 'Cards Sandbox Payment Pointer',
+      walletAddressService: this.walletAddressService
+    })
+
+    const requestBody: ICreateCustomerRequest = {
+      walletAddress: account.gateHubWalletId,
+      account: {
+        productCode: this.env.GATEHUB_ACCOUNT_PRODUCT_CODE,
+        currency: 'EUR',
+        card: {
+          productCode: this.env.GATEHUB_CARD_PRODUCT_CODE
+        }
+      },
+      nameOnCard: this.env.GATEHUB_NAME_ON_CARD,
+      citizen: {
+        name: userState.profile.first_name,
+        surname: userState.profile.last_name
+      }
+    }
+
+    const customer = await this.gateHubClient.createCustomer(
+      managedUserId,
+      requestBody
+    )
+    const customerId = customer.customers.id!
+
+    await User.query().findById(userId).patch({
+      customerId
+    })
+
+    await this.gateHubClient.updateMetaForManagedUser(managedUserId, {
+      paymentPointer: paymentPointer.url,
+      customerId
+    })
```

Comment:
Do we still need to update the meta in this case?

Reply:
I also cannot think of a use case in the future where we would need these fields on the sandbox environment.

Reply:
Yeah, lets remove it for now.